Readme_Kor.md

```diff
@@ -0,0 +1,419 @@
+# scrcpy (v1.10)
+
+This document will be updated frequently along with the original Readme file
+이 문서는 원어 리드미 파일의 업데이트에 따라 종종 업데이트 될 것입니다
+
+ 이 어플리케이션은 UBS ( 혹은 [TCP/IP][article-tcpip] ) 로 연결된 Android 디바이스를 화면에 보여주고 관리하는 것을 제공합니다.
+ _GNU/Linux_, _Windows_ 와 _macOS_ 상에서 작동합니다.
+ (아래 설명에서 디바이스는 안드로이드 핸드폰을 의미합니다.)
+ 
+[article-tcpip]:https://www.genymotion.com/blog/open-source-project-scrcpy-now-works-wirelessly/
+
+![screenshot](https://github.com/Genymobile/scrcpy/blob/master/assets/screenshot-debian-600.jpg?raw=true)
+
+주요 기능은 다음과 같습니다.
+
+ - **가벼움** (기본적이며 디바이스의 화면만을 보여줌)
+ - **뛰어난 성능** (30~60fps)
+ - **높은 품질** (1920×1080 이상의 해상도)
+ - **빠른 반응 속도** ([35~70ms][lowlatency])
+ - **짧은 부팅 시간** (첫 사진을 보여주는데 최대 1초 소요됨)
+ - **장치 설치와는 무관함** (디바이스에 설치하지 않아도 됨)
+
+[lowlatency]: https://github.com/Genymobile/scrcpy/pull/646
+
+
+## 요구사항
+
+안드로이드 장치는 최소 API 21 (Android 5.0) 을 필요로 합니다.
+
+디바이스에 [adb debugging][enable-adb]이 가능한지 확인하십시오.
+
+[enable-adb]: https://developer.android.com/studio/command-line/adb.html#Enabling
+
+어떤 디바이스에서는, 키보드와 마우스를 사용하기 위해서 [추가 옵션][control] 이 필요하기도 합니다.
+
+[control]: https://github.com/Genymobile/scrcpy/issues/70#issuecomment-373286323
+
+
+## 앱 설치하기
+
+
+### Linux (리눅스)
+
+리눅스 상에서는 보통 [어플을 직접 설치][BUILD] 해야합니다. 어렵지 않으므로 걱정하지 않아도 됩니다.
+
+[BUILD]:https://github.com/Genymobile/scrcpy/blob/master/BUILD.md
+
+[Snap] 패키지가 가능합니다 : [`scrcpy`][snap-link].
+
+[snap-link]: https://snapstats.org/snaps/scrcpy
+
+[snap]: https://en.wikipedia.org/wiki/Snappy_(package_manager)
+
+Arch Linux에서, [AUR] 패키지가 가능합니다 : [`scrcpy`][aur-link].
+
+[AUR]: https://wiki.archlinux.org/index.php/Arch_User_Repository
+[aur-link]: https://aur.archlinux.org/packages/scrcpy/
+
+Gentoo에서 ,[Ebuild] 가 가능합니다 : [`scrcpy/`][ebuild-link].
+
+[Ebuild]: https://wiki.gentoo.org/wiki/Ebuild
+[ebuild-link]: https://github.com/maggu2810/maggu2810-overlay/tree/master/app-mobilephone/scrcpy
+
+
+### Windows (윈도우)
+
+윈도우 상에서, 간단하게 설치하기 위해 종속성이 있는 사전 구축된 아카이브가 제공됩니다 (`adb` 포함) :
+
+ - [`scrcpy-win32-v1.10.zip`][direct-win32]  
+   _(SHA-256: f98b400b3764404b33b212e9762dd6f1593ddb766c1480fc2609c94768e4a8e1)_
+ - [`scrcpy-win64-v1.10.zip`][direct-win64]  
+   _(SHA-256: 95de34575d873c7e95dfcfb5e74d0f6af4f70b2a5bc6fde0f48d1a05480e3a44)_
+
+[direct-win32]: https://github.com/Genymobile/scrcpy/releases/download/v1.10/scrcpy-win32-v1.10.zip
+[direct-win64]: https://github.com/Genymobile/scrcpy/releases/download/v1.10/scrcpy-win64-v1.10.zip

```

Comment:
Could you just link to the english version of this section instead, to avoid updating all the localized README on new release, please?

Reply:
I have changed the Windows installation section with the link! Also added a note about it :)

Reply:
Thank you :+1: